chapter_01_domain_model.asciidoc

```diff
@@ -65,7 +65,7 @@ and cannot encourage you enough to go on and read a proper DDD book.
 
 * The original https://domainlanguage.com/ddd/["blue book"],
   _Domain-Driven Design_ by Eric Evans (Addison-Wesley, 2003)
-* Or, some people prefer the "red book", _Implementing Domain-Driven Design_,
+* Or, some people prefer the https://learning.oreilly.com/library/view/implementing-domain-driven-design/9780133039900/["red book"], _Implementing Domain-Driven Design_,
```

Comment:
OR will probably have an opinion on this, but i think they'll want us to use formal citation syntax plus a bibliography for these.  will merge it in the meantime...